### ISSUE TITLE
Support for 6.72

### ISSUE THREAD

**Issue by Anuskuss** [Zer0xFF/reactPSPLUS#3]
Can you update this to work for FW 6.72? Also, would be pretty nice to be able to configure it so we can set the date/time to any date :)

**Comment by Zer0xFF** [comment]
I dont have a PS4 on 6.72, so i wont be updating this, anyone interested is free to send a PR

**Comment by neo305** [comment]
Awwww man someone please update this to 6.72 :)

**Comment by psychowood** [comment]
Does this need [ps4-kernel-clock](https://github.com/Scene-Collective/ps4-kernel-clock) to work?

I've got a 6.72 at hand, I'd try to test it, but I don't know where to start :)

**Comment by Anuskuss** [comment]
@psychowood The dev claims that it works, but it doesn't for me. Can you [send the payload to your PS4](https://www.youtube.com/watch?v=IaS9R_s2R3Y) and verify?

**Comment by psychowood** [comment]
@Anuskuss where did you read that claim? Afaik there are changes needed, or at least a rebuild for 6.72, Zero0xFF asked for a PR...

**Comment by Anuskuss** [comment]
@psychowood Scene-Collective/ps4-kernel-clock@0a093756ac80b44ad0e319f5cdf41ef2cf42b032

**Comment by psychowood** [comment]
> @psychowood Scene-Collective/ps4-kernel-clock@0a093756ac80b44ad0e319f5cdf41ef2cf42b032

Yep, this needs at least a rebuild, I don't understand why you are saying this

> The dev claims that it works

**Comment by neo305** [comment]
anybody try this on 9.00 yet?
